examples.json

```
@@ -0,0 +1,347 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "description": "This is an example bookmarks server.",
+    "version": "1.0.0",
+    "title": "EBSCO API GUIDE",
+    "contact": { "email": "iquattlebaum@ebsco.com" }
+  },
+  "host": "bookmarks.swagger.io",
+  "basePath": "/v2",
+  "tags": [
+    {
+      "name": "bookmarks",
+      "description": "All the bookmarks..."
+    }
+  ],
+  "schemes": ["https", "http"],
+  "paths": {
+    "/bookmarks": {
+      "parameters": [
+        {
+          "$ref": "#/parameters/BearerHeader"
+        },
+        {
+          "$ref": "#/parameters/PersonalizedHeader"
+        }
+      ],
+      "get": {
+        "tags": ["bookmarks"],
+        "summary": "Find bookmarks",
+        "description": "Returns bookmarks",
+        "produces": ["application/json"],
+        "parameters": [
+          {
+            "name": "page",
+            "in": "query",
+            "description": "Integer of page to return",
+            "required": false,
+            "type": "integer"
+          },
+          {
+            "name": "page_size",
+            "in": "query",
+            "description": "Integer of number of items in result page",
+            "required": false,
+            "type": "integer"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "bookmarks": {
```

Just a note -- if we want to standardize on `items` per the apparent v2 Paypal API spec, we'd want to change this here, too.
```suggestion
                "items": {
```

---

fair enough... I think they may have it wrong in their own patterns.md file them... https://github.com/paypal/api-standards/blob/master/patterns.md#page-navigation notice plans

---

done

---

Yeah they absolutely have their wires crossed in their own docs (first noticed that in a PR I opened up over here, if you wanna take a look: https://github.com/EBSCOIS/api-standards/pull/12)